### ISSUE TITLE
Images with transparency not looking correct when opening PPTX

### ISSUE THREAD

**Issue by TimonUhr** [ONLYOFFICE/DocumentServer#1863]

**Comment by SamanthaWabi** [comment]
Is there any update on this or a projected timeline for a fix?? I'm experiencing the same thing with watermarks showing up as solid black instead of the opacity I've set.

**Comment by Rita-Bubnova** [comment]
@SamanthaWabi, Sorry, no update.
You can contact sales team sales@onlyoffice.com to speed up things.

**Comment by anton-degterev** [comment]
In my opinion, at the moment this is one of the main problems that prevents the massive use of ONLYOFFICE Presentation Editor as a replacement for Microsoft Office PowerPoint.

**Comment by ThePumba** [comment]
Yes, this, and the Issue #1862, 1 1/2 years ago. I wanted to switch to ONLYOFFICE, but since these issues are still there, I will stay with MS Office. Interestingly, in [this](https://www.onlyoffice.com/blog/2017/01/11-things-you-can-do-with-onlyoffice-but-cant-with-google-docs-or-ms-word-online?from=comparison) post from 2017, they make fun of MS Office Online and Google Docs for sometimes messing with the formatting and claim, 'ONLYOFFICE will retain the original formatting of the document regardless of the file size and complexity of its elements. Don’t forget what your document looked like!' However, issues like this one still exist seven years later.

**Comment by yllekz** [comment]
Can confirm this is happening as well. LibreOffice displays a low-opacity image correctly, office.com displays it correctly, OnlyOffice does not.
